internal/runners/packages/install.go

```diff
@@ -16,26 +12,16 @@ type InstallRunParams struct {
 
 // Install manages the installing execution context.
 type Install struct {
-	cfg  configurable
-	out  output.Outputer
-	proj *project.Project
-	prompt.Prompter
-	auth *authentication.Auth
+	prime primeable
```

Comment:
We should normally avoid doing this, but in this case passing `prime` along to the package operation saves us a lot of useless boilerplate.